### ISSUE TITLE
Bug report: dde-launcher don't work after last upgrade

### ISSUE THREAD

**Issue by PrzemekSkw** [linuxdeepin/developer-center#2991]
Hello, after last upgrade I cannot start dde-launcher. I see only some blurred space where it should be.

```
$ inxi -F
System:    Host: matebook-PC Kernel: 5.10.101-amd64-desktop x86_64 bits: 64 Desktop: Deepin 20.4 
           Distro: Deepin 20.4 
Machine:   Type: Laptop System: HUAWEI product: KLVL-WXX9 v: M1300 serial: <superuser required> 
           Mobo: HUAWEI model: KLVL-WXX9-PCB v: M1300 serial: <superuser required> UEFI: HUAWEI v: 1.06 
           date: 09/14/2020 
Battery:   ID-1: BAT0 charge: 22.1 Wh condition: 56.3/54.9 Wh (102%) 
CPU:       Info: 6-Core model: AMD Ryzen 5 4600H with Radeon Graphics bits: 64 type: MT MCP L2 cache: 3 MiB 
           Speed: 1397 MHz min/max: 1400/3000 MHz Core speeds (MHz): 1: 1397 2: 1397 3: 1353 4: 1234 5: 1397 
           6: 1397 7: 1400 8: 1397 9: 1474 10: 1636 11: 1348 12: 1265 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] Renoir driver: amdgpu v: kernel 
           Device-2: IMC Networks type: USB driver: uvcvideo 
           Display: x11 server: X.Org 1.20.4 driver: loaded: amdgpu,ati unloaded: fbdev,modesetting,vesa 
           resolution: 2160x1440~60Hz 
           OpenGL: renderer: AMD RENOIR (DRM 3.40.0 5.10.101-amd64-desktop LLVM 11.0.1) v: 4.6 Mesa 20.3.5 
Audio:     Device-1: Advanced Micro Devices [AMD/ATI] driver: snd_hda_intel 
           Device-2: Advanced Micro Devices [AMD] Raven/Raven2/FireFlight/Renoir Audio Processor driver: N/A 
           Device-3: Advanced Micro Devices [AMD] Family 17h HD Audio driver: snd_hda_intel 
           Sound Server: ALSA v: k5.10.101-amd64-desktop 
Network:   Device-1: Realtek RTL8822CE 802.11ac PCIe Wireless Network Adapter driver: rtw_8822ce 
           IF: wlp1s0 state: up mac: 28:cd:c4:b7:cd:d7 
Bluetooth: Device-1: type: USB driver: btusb 
           Report: ID: hci0 state: up running pscan iscan bt-v: 3.0 address: 28:CD:C4:B7:CD:D8 
Drives:    Local Storage: total: 476.94 GiB used: 175.54 GiB (36.8%) 
           ID-1: /dev/nvme0n1 vendor: Western Digital model: PC SN530 SDBPNPZ-512G-1027 size: 476.94 GiB 
Partition: ID-1: / size: 38.43 GiB used: 17.89 GiB (46.5%) fs: ext4 dev: /dev/nvme0n1p4 
           ID-2: /home size: 284.95 GiB used: 124.54 GiB (43.7%) fs: ext4 dev: /dev/nvme0n1p9 
Swap:      ID-1: swap-1 type: partition size: 7.45 GiB used: 1.3 MiB (0.0%) dev: /dev/nvme0n1p8 

```
I have dde-launcher version: 5.5.6-1
DeepinOS: (community) 20.4

dmesg:
https://pastebin.com/Nc3mDs7T

Regards,
Przemek

**Comment by PrzemekSkw** [comment]
I have upgrade now to 20.5 and dd-launcher works OK.

ps. I have also wifi icon with `6`. Is it possible that I have wifi6 now if my laptop (Huawei Matebook 14 - 2021) was only wifi5 compatibile? I see now speedtest in web and it works like wifi6 (I compare to my smartphone), before upgrade it was slower. How this is possible? 
Regards.

**Comment by BLumia** [comment]
> I have upgrade now to 20.5 and dd-launcher works OK.

Since it's resolved on deepin v20.5 `dde-launcher 5.5.11-1`, I'll close this issue now :)

> Is it possible that I have wifi6 now if my laptop (Huawei Matebook 14 - 2021) was only wifi5 compatibile?

I'm not an expert on WIFI6 but I'm afraid it's not possible. btw for general discussion, please consider [creating a discussion thread in the discussion board](https://github.com/linuxdeepin/developer-center/discussions) :)
